docs/troubleshooting-guide.md

```diff
@@ -0,0 +1,87 @@
+# OPCT - Troubleshooting Guide
```

Comment:
After re-reading this document, I'm not sure its needed. The content is conflicting because it explains troubleshooting test failures (covered in support-guide.md already) and the other content is covering failure of the certification tool itself. 

I think we should either remove this document entirely or re-focus it on a single subject.